src/utils/trust/details.mapper.ts

```diff
@@ -0,0 +1,99 @@
+import { ApplicationData } from '../../model';
+import { Trust, TrustBeneficialOwner, TrustKey } from '../../model/trust.model';
+import * as Page from '../../model/trust.page.model';
+import { BeneficialOwnerIndividual, BeneficialOwnerIndividualKey } from '../../model/beneficial.owner.individual.model';
+import { BeneficialOwnerOther, BeneficialOwnerOtherKey } from '../../model/beneficial.owner.other.model';
+import { addTrustToBeneficialOwner, removeTrustFromBeneficialOwner } from '../../utils/trusts';
+import { BeneficialOwnerTypeChoice } from '../../model/beneficial.owner.type.model';
+
+//  to page form mapping
+const mapDetailToPage = (
+  appData: ApplicationData,
+  trustId: string,
+): Page.TrustDetails => {
+  const trustData = appData[TrustKey]?.find(trust => trust.trust_id === trustId) ?? {} as Trust;
+
+  const trustBo: string[] =
```

Comment:
Potentially contains several Beneficial Owners by the looks of it - would rename to 'trustBos'.

Reply:
renamed